docs/concepts/09_glossary.md

```diff
@@ -0,0 +1,157 @@
+---
+id: glossary
+title: KILT Glossary
+---
+
+Here is a glossary of terms related to the KILT Protocol:
+
+## W3C: Self-Sovereign Identity (SSI)
+
+### Decentralized Identifiers (DID)
+
+Stands for Decentralized Identifier, a unique identifier for entities (people, organizations, or things) that can be anchored to a blockchain and provides verifiable digital identity.
+For example, **did:kilt:4sxSYXakw1ZXBymzT9t3Yw91mUaqKST5bFUEjGEpvkTuckar**.
+In KILT, identity is built by obtaining credentials linked to the DID.
+A DID can be used for people, machines, services and anything that identities can be built on.
+
+### DID Authentication
+
+DID authentication is the process of proving that a DID owner has control over a DID, typically by using a digital signature or other cryptographic mechanism.
+
+### DID Communication
+
+DID communication refers to the use of DIDs to enable secure and decentralized communication between two or more parties, without the need for a central intermediary.
+
+### DID Controller
+
+A DID controller is an entity that has control over a DID, such as the entity that created the DID or an entity that has been authorized by the DID owner to manage the DID.
+
+### DID Document
+
+A DID document is a JSON-LD document that contains information about a DID, including public keys, service endpoints, and other metadata.
+
+### DID Method
+
+A DID method is a set of rules and specifications for how a DID is created, resolved, and managed on a particular network or platform.
+
+### DID Resolver
+
+A DID resolver is a software component that can resolve a DID to a DID Document, which contains information about the DID, such as public keys, service endpoints, and other metadata.
+
+### Self-Sovereign Identity
+
+A decentralized digital identity management system that enables individuals to own and control their identity information using secure digital technologies, such as blockchain.
+SSI eliminates the need for intermediaries and provides individuals with greater privacy, security, and control over their personal data.
+It is an emerging concept that has the potential to transform how identity is managed and verified across various sectors.
+
+### Verifiable Credentials
+
+Verifiable credentials are digital credentials that can be used to prove claims about a person, organization, or thing, and are designed to be portable, interoperable, and privacy-preserving.
+Verifiable credentials are often associated with DIDs and can be stored and managed using a DID-based identity system.
+
+## KILT Protocol Specific Terms
+
+### Attestation
+
+The act of providing evidence or confirming the validity of a claim, typically performed by a trusted attester.
+
+### Attester
+
+A person, organization or entity that verifies and validates the identity or credentials of an individual in a digital or physical setting.
```

Comment:
And they verify the information. I think you are separating the two too much. I would argue that the attester verifies information as well as validates it. Verify in this context is to make sure the data is correct. 

The attester stamps that they have done this as the attestation. The verifier takes that trusted stamp of approval and checks by verifying the data integrate to ensure the data matches but uses the attestation as a bridge of trust.

Reply:
But their main task is to issue the attestation and for that they need to verify the data. I would prefer if we could stress the point of attesting, rather than the verifying point. For someone not knowing anything about this it might be difficult to spot the difference between verifiers and entities that verify.

Reply:
```suggestion
An entity makes sure that certain statements are accurate. This entity examines claims made by other entities and only approves those that are true, issuing a certification. However, there may be dishonest attesters who approve false statements as well.
```

Reply:
How does this sound?

Reply:
sounds good